cinema/serializers.py

```diff
@@ -53,12 +75,9 @@
 
 class MovieSessionListSerializer(MovieSessionSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
-    cinema_hall_name = serializers.CharField(
-        source="cinema_hall.name", read_only=True
-    )
-    cinema_hall_capacity = serializers.IntegerField(
-        source="cinema_hall.capacity", read_only=True
-    )
+    cinema_hall_name = serializers.CharField(source="cinema_hall.name", read_only=True)
+    cinema_hall_capacity = serializers.IntegerField(source="cinema_hall.capacity", read_only=True)
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `MovieSessionListSerializer` correctly includes additional fields like `movie_title`, `cinema_hall_name`, and `cinema_hall_capacity` to provide a more detailed representation of a movie session.